html/js/tree/Move.js

```diff
@@ -0,0 +1,34 @@
+GTE.TREE = (function (parentModule) {
+    "use strict";
+
+    /**
+    * Creates a new Move.
+    * @class
+    * @param {Node} parent Parent node origin of the move.
+    * @param {Node} child Child node destiny of the move.
+    */
+    function Move (parent, child) {
+        this.parent = parent;
+        parent.children[child] = this;
+        this.child = child;
+        this.line = {};
+    }
+
+    /**
+    * ToString function
+    */
+    Move.prototype.toString = function moveToString() {
```

Comment:
Does the moveToString function name have any purpose? If not, I  would keep it anonymous for consistency with everything else.